apisix/admin/resource.lua

```diff
@@ -49,7 +49,41 @@ local function split_typ_and_id(id, sub_path)
 end
 
 
-function _M:check_conf(id, conf, need_id, typ)
+local function check_create_update_time(conf)
+    local not_allow_create_time = "forbidden create_time in request body"
+    local not_allow_update_time = "forbidden update_time in request body"
+
+    if conf.create_time then
+        return not_allow_create_time
+    end
```

Comment:
Is it possible to define a check list and sequentially check for properties that should not exist? like
```
local forbidden_properties = {create_time, update_time}
```

Reply:
great! I will fix it later